docker/node.dockerfile

```diff
@@ -16,8 +19,8 @@ RUN yarn run vendor:clean
 RUN yarn run vendor:setup
 
 # Build pocketdex
-WORKDIR /app
-RUN yarn codegen && yarn build
+RUN #NODE_ENV=$NODE_ENV yarn run codegen
```

Comment:
Am I interpreting this correctly as commented out? It seems like we shouldn't need it since `yarn build` runs `yarn codegen`.
```suggestion
```

Reply:
My bad you are right, that line should be removed, because the next one which is `RUN NODE_ENV=$NODE_ENV yarn run build` does both `codegen` & `build`